test/JDBC/expected/SYSTEM_USER-vu-verify.out

```diff
@@ -0,0 +1,31 @@
+-- tsql
+SELECT session_user, system_user, current_user, db_name();
+GO
+~~START~~
+nvarchar#!#nvarchar#!#varchar#!#nvarchar
+dbo#!#jdbc_user#!#dbo#!#master
+~~END~~
+
+
+-- tsql user=system_user_vu_prepare_r1 password=123
+SELECT session_user, system_user, current_user, db_name();
+GO
+~~START~~
+nvarchar#!#nvarchar#!#varchar#!#nvarchar
+guest#!#system_user_vu_prepare_r1#!#guest#!#master
+~~END~~
+
+
+-- tsql user=system_user_vu_prepare_r2 password=123
+SELECT session_user, system_user, current_user, db_name();
+GO
+~~START~~
+nvarchar#!#nvarchar#!#varchar#!#nvarchar
+guest#!#system_user_vu_prepare_r2#!#guest#!#master
+~~END~~
+
+
+-- tsql
+DROP LOGIN system_user_vu_prepare_r1;
```

Comment:
It would be better to have this part in a separate cleanup file

Reply:
Separated this part in a cleanup file.